pyproject.toml

```diff
@@ -81,7 +81,7 @@ all = [
     "beautifulsoup4",
     "html5lib",
     "bleach",
-    "pandas",
+    "pandas>=2.0",
```

Comment:
Do we need a change log? 🤔

Reply:
Do we have one for optional dependency bumps following our support cadence? We should maybe standardize this decision and update our what's new template to reflect this?

Reply:
I am not sure but suddenly going from no pin to 2.0 might surprise some users. I think `pandas` is popular enough nowadays that we should probably announce the pin? What do you think, @taldcroft ?

Reply:
Ping @taldcroft.

Reply:
It might be a good idea here to add a changelog entry esp. since this is a major version update for pandas.